core/conversion/converters/impl/element_wise.cpp

```diff
@@ -25,6 +25,22 @@ nvinfer1::ITensor* clamp_util(
   return clamp_layer_out;
 }
 
+bool two_tensor_input_single_elment_wise_layer(
```

Comment:
Feel like I'd rather this be a macro similar to unary that just takes the torchscript schema and the enum.